### PR TITLE
feat(notifications): added notifications

### PR DESCRIPTION
**Pull request by serafim-san**
## Summary
Added notifications

## Notion card
[New design](https://www.notion.so/santiment/Library-Flag-Messages-updates-74a14adf864345f79e14f095d24935e4)
[Old one](https://www.notion.so/santiment/Notifications-implementation-using-Svelte-5-san-webkit-next-90c54ddec7804787b70adebdc573696f)

## Screenshots
image
image
image
image

### CODE REVIEW

File: src/lib/ui/app/PaymentForm/flow.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Query } from '$lib/api/executor.js'
 import { useStripeCtx } from '$lib/ctx/stripe/index.js'
-import { notifcation } from '$ui/core/Notifications/index.js'
+import { notification } from '$ui/core/Notifications/index.js'
```
Comment (serafim-san) [inline src/lib/ui/app/PaymentForm/flow.ts:3]: removed typo in `notification` object

File: src/lib/ui/core/Notifications/index.ts [hunk 2]
```diff
@@ -1,2 +1,34 @@
...
-export { toast as notifcation } from 'svelte-sonner'
+import Component from './Notification.svelte'
+import type { ComponentProps, ComponentType } from 'svelte'
+
+function createToast(
+  icon: string,
+  message: string,
+  options?: Omit<ComponentProps<Component>, 'icon' | 'message'>,
+) {
+  return toast.custom(Component as unknown as ComponentType, {
```
Comment (serafim-san) [inline src/lib/ui/core/Notifications/index.ts:12]: I had to do this because of the different types of components in Svelte 4 and Svelte 5

Comment (serafim-san) [reply src/lib/ui/core/Notifications/index.ts:12]: Here's the error:
```
Argument of type '__sveltets_2_IsomorphicComponent<Props, { closeToast: CustomEvent<any>; } & { [evt: string]: CustomEvent<any>; }, {}, {}, "">' is not assignable to parameter of type 'ComponentType'.
  Type '__sveltets_2_IsomorphicComponent<Props, { closeToast: CustomEvent<any>; } & { [evt: string]: CustomEvent<any>; }, {}, {}, "">' is not assignable to type 'new (options: ComponentConstructorOptions<Record<string, any>>) => SvelteComponent<Record<string, any>, any, any>'.
    Types of parameters 'options' and 'options' are incompatible.
      Type 'ComponentConstructorOptions<Record<string, any>>' is not assignable to type 'ComponentConstructorOptions<Props>'.
        Type 'Record<string, any>' is missing the following properties from type 'Props': icon, messagets(2345)
```

File: src/lib/ui/core/Dialog/vaul.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
 import { getAllContexts } from 'svelte'
 
-export const getVaulCtx = (ctx = getAllContexts()): [] | [Symbol, any] =>
-  Array.from(ctx.entries()).find(([key, value]) => value?.states?.drawerId) || []
+export const getVaulCtx = (ctx = getAllContexts()): [] | [symbol, any] =>
+  Array.from(ctx.entries()).find(([_key, value]) => value?.states?.drawerId) || []
```
Comment (serafim-san) [inline src/lib/ui/core/Dialog/vaul.ts:4]: It was changed because eslint error. `Symbol` letter case changed automatically by `eslint --fix`

**Review by DmitriVanGuard — APPROVED** [review]

**Pull request merged**